### PR TITLE
Fix css class conflicts

### PR DESCRIPTION
**Pull request by tomas-stefano**
On mounted apps that use the DfE autocomplete the `GovUK all` replaces any customization of the `GovUK base` which is not the desired output.

Importing only what's needed can decrease the level of friction with any apps that use the library.

Also wrapped the CSS under the suggestions class so it also doesn't cause conflicts with anything on another app. The name 'suggestions' will be changed in the future.

### CODE REVIEW

**Comment by duncanjbrown** [comment]
I suppose it's still possible that we could import something in govuk/base which affects an override? Could we get away without the dependency on `govuk-frontend` at all?

**Comment by tomas-stefano** [comment]
> I suppose it's still possible that we could import something in govuk/base which affects an override? 
Yes, it is still possible but not likely as the govuk/all. I tested locally and I couldn't catch any conflict yet.

> Could we get away without the dependency on `govuk-frontend` at all?
Maybe we could but I don't know yet. Maybe someone can help if we're in hurry? @frankieroberto do you know anything about this?

**Comment by frankieroberto** [comment]
The autocomplete component doesn’t follow or depend upon `govuk-frontend` at all (as it predates it and hasn't been updated) so we need to include the govuk-frontend styles somewhere in order to get the fonts to match.

It's all a bit of a mess... 😬  I'll have a think about how we might do this in the least painful way (and I'll ask @edwardhorsford too).

**Review by duncanjbrown — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -120,8 +120,10 @@ Supported options:
 * `minLength`: The minimum number of characters that should be entered before the autocomplete will attempt to suggest options. When the query length is under this, the aria status region will also provide helpful text to the user informing them they should type in more.
 * `rawAttribute`: the second parameter that will be sent when user submits the form. This is useful when the user types free text instead of choosing any element in the collection.
 
+## Stylesheet
+
 If you need to import styles as well:
```
Comment (duncanjbrown) [inline README.md:125]: We could say this is required (I think it is!)

**Pull request merged**